src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -254,15 +255,17 @@ def _upload_resources(clients: UploadClients, upload_state: UploadState) -> None
         media_previously_ingested=upload_state.config.media_previously_uploaded,
     )
 
+    progress_bar = tqdm(upload_state.pending_resources.copy(), leave=False)
     try:
-        for creation_attempts_of_this_round, resource in enumerate(upload_state.pending_resources.copy()):
+        for creation_attempts_of_this_round, resource in enumerate(progress_bar):
             _upload_one_resource(
                 upload_state=upload_state,
                 resource=resource,
                 ingest_client=clients.asset_client,
                 resource_create_client=resource_create_client,
                 creation_attempts_of_this_round=creation_attempts_of_this_round,
             )
+            progress_bar.set_description(f"Creating Resources (failed: {len(upload_state.failed_uploads)})")
```

Comment:
After how much time is this displayed?
The user may want to stop the upload as soon as one image fails...

Reply:
this is updated each loop, so ideally it will always display `Creating Resources (failed: 0)`, but if something goes wrong it should update pretty much immediately

Reply:
in case you're wondering how it looks:

![image](https://github.com/dasch-swiss/dsp-tools/assets/33053745/cf54b411-3017-4161-b476-e08138fb2890)